src/client/DeploymentTable.tsx

```diff
@@ -35,8 +40,38 @@ interface ITableProps {
 }
 
 let releasesUrl: string = "";
+let tableItems: ObservableArray<IDeploymentField>;
+let rawTableItems: IDeploymentField[];
+let columns: Array<ITableColumn<IDeploymentField>>;
+let sortFunctions: Array<
+  ((item1: IDeploymentField, item2: IDeploymentField) => number) | null
+>;
+
+const sortingBehavior = new ColumnSorting<IDeploymentField>(
+  (
+    columnIndex: number,
+    proposedSortOrder: SortOrder,
+    event: React.KeyboardEvent<HTMLElement> | React.MouseEvent<HTMLElement>
+  ) => {
+    tableItems.splice(
```

Comment:
Can we use `tableItems = tableItems.slice(` instead? In general we should prefer immutable transforms (don't know if `ObservableArray` requires mutable transforms -- if so, i'm fine with leaving as is)

Reply:
Observable array does not support slice, only splice.